bin/main/application.yml

```
@@ -0,0 +1,49 @@
+kakao:
+  native-app-key: ${KAKAO_SECRET_KEY}
+  redirect-uri: http://localhost:8080/login/oauth2/code/kakao    # 인가 코드
+  token-uri: https://kauth.kakao.com/oauth/token      # 토큰 발급 요청
+  user-info-uri: https://kapi.kakao.com/v2/user/me    # 유저 정보 조회
+
+
+colsms:
+  api:
+    key: ${SMS_API_KEY}
+    secret: ${SMS_API_SECRET}
+    sender: 01087164428
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)